validator/htmlparser/css/parse-css-urls_test.cc

```diff
@@ -0,0 +1,164 @@
+//
+// Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the license.
+//
+
+#include "css/parse-css-urls.h"
+
+#include <ostream>
+#include "glog/logging.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "absl/strings/str_cat.h"
+#include "strings.h"
+
+using testing::Eq;
+using testing::Pointwise;
+
+namespace htmlparser::css::url {
+
+std::string CssSegment_Type_Name(url::CssSegment::Type type) {
+  switch (type) {
+    case url::CssSegment::BYTES:
+      return "BYTES";
+    case url::CssSegment::IMAGE_URL:
+      return "IMAGE_URL";
+    case url::CssSegment::OTHER_URL:
+      return "OTHER_URL";
+  }
+}
+
+// This is so that gunit/gmock print reasonable errors in case a
+// CssSegmentEq match fails. Must be defined in the
+// quality_dni::parse_css_urls namespace, like url::CssSegment itself. See
+// go/gunitadvanced#teaching-googletest-how-to-print-your-values
```

Comment:
Maybe remove / fix this comment so it makes more sense in github.

Reply:
Removed the comment inline in this pull.